config/puma/production.rb

```diff
@@ -19,6 +19,19 @@
 #
 workers ENV.fetch('FOREMAN_PUMA_WORKERS', 2).to_i
 
+# Puma worker timeout
+#
+# Verifies that all workers have checked in to the master process within
+# the given timeout. If not the worker process will be restarted. This is
+# not a request timeout, it is to protect against a hung or dead process.
+# Setting this value will not protect against slow requests.
+#
+# The minimum value is 6 seconds, the default value is 60 seconds
```

Comment:
I'm not sure I like a verbatim copy of the upstream documentation since it can go out of sync. I'd actually suggest something like.
```suggestion
# Puma worker timeout. It's not recommended to change this since it can mask real bugs.
```